bidwire/tests/test_boston_notice_scraper.py

```diff
@@ -0,0 +1,46 @@
+import os
+import csv
+from lxml import html
+from urllib.parse import urljoin
+from scrapers.boston_notice_scraper import BostonNoticeScraper
+from . import common
+
+
+class TestBostonNoticeScraper(object):
+    session = None
+    page_str = ""
+    notice_strs = {}
+    expected_docs = []
+
+    def test_scrape(self):
+        scraper = BostonNoticeScraper()
+
+        def scrape_desc(href):
+            try:
+                return scraper.scrape_desc_page(self.notice_strs[href])
+            except KeyError:
+                return
+        scraper.scrape_desc = scrape_desc
+
+        assert [
+            [doc.url, doc.title] for doc in
+            scraper.scrape_notices_page_and_notices(self.session, html.fromstring(self.page_str))
+        ] == self.expected_docs
```

Comment:
It would be nice if this test were testing `test_scrape`, including the DB save. Especially since there's a model change here it would be good to ensure the DB operations.